example_inference.ipynb

```diff
@@ -125,17 +89,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": 4,
    "id": "902c1057",
    "metadata": {},
    "outputs": [],
    "source": [
-    "res = img2mol(filepath=\"examples/digital_example1.png\", cddd_server=cddd_server)"
+    "res = img2mol(filepath=\"examples/digital_example1.png\")"
```

Comment:
Most changes in the notebook are basically related to running this with the local_cddd. I would recommend removing the changes of the example_inference.ipynb from this PR, and create a second example_inference_local_cddd.ipynb with an example and mention the difference in the README.

Reply:
There are now two versions of the notebook. The difference is mentioned in the README